packages/wallet/backend/src/walletAddress/service.ts

```diff
@@ -387,6 +345,7 @@ export class WalletAddressService implements IWalletAddressService {
 
     if (transfer.status?.status !== 'SUCCESS') {
       if (type === 'OUTGOING') {
+        //TODO this might not be needed
```

Comment:
as in debt being part of the walletAddress model?

Reply:
Yes, but I'm not 100% sure we should delete it

Reply:
The point you are raising is valid. I think there should be a retry mechanism in place.

But I think the retry mechanism can be part of the current flow:
Transfer to rapyd fails -> transfer should be picked up by the next iteration of the cron-job and retried, since it should again be detected as imbalance.

This way, no additional mechanism requiring debt would be needed.